### PR TITLE
Tweaks to policy that excludes files from being uploaded

### PR DESCRIPTION
**Pull request by gacarini**

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: OneDrive/use-group-policy.md [hunk 1]
```diff
@@ -252,11 +252,13 @@ Users will also see a message in the OneDrive activity center that explains why
 
 If you disable or do not configure this setting, all supported files in all synced folders will be uploaded. 
 
-Enabling this policy sets the following registry key value to 1
-HKLM\SOFTWARE\Policies\Microsoft\OneDrive\EnableODIgnoreListFromGPO ="dword:00000001"
+Enabling this policy creates a list of strings under the follow following path HKLM\SOFTWARE\Policies\Microsoft\OneDrive\EnableODIgnoreListFromGPO.
```
Comment (JohanFreelancer9) [inline OneDrive/use-group-policy.md:255]:
```suggestion
Enabling this policy creates a list of strings under the following path: HKLM\SOFTWARE\Policies\Microsoft\OneDrive\EnableODIgnoreListFromGPO.
```

File: OneDrive/use-group-policy.md [hunk 2]
```diff
@@ -252,11 +252,13 @@ Users will also see a message in the OneDrive activity center that explains why
...
 
 > [!NOTE]
 > This setting gives you more flexibility than the [Block syncing of specific file types setting](block-file-types.md) in the admin center. Also with this setting, users don't see errors for the excluded files.
+
+> [!NOTE]
+> This setting does not support excluding Office files from being uploaded. All other file types are supported.
```
Comment (JohanFreelancer9) [inline OneDrive/use-group-policy.md:261]:
```suggestion
> This setting does not support excluding Office files from being uploaded. Excluding all other file types are supported.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by scanum** [comment]
@kaarins PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**